### PR TITLE
delete note

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: src/pages/archieve-page/ArchieveNote.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const ArchivedNote = () => {
+    return(
+    <div>
+      <Navbar />
+      <div className={styles.container}>
+        <div>
+          <SideBar />
+        </div>
+          <div className={styles.saveNoteContainer}>
+              <ArchiveNoteContainer />
```
Comment (sanjitsarkar) [inline src/pages/archieve-page/ArchieveNote.jsx:14]: You can use map here instead

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and easy to understand

File: src/pages/homepage/HomePage.jsx [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-import { useNote } from "../../context";
+import { useAuth, useTrash, useNote } from "../../context";
 import styles from "./HomePage.module.css";
+import { trashNoteHandler } from "../../service";
+
 const HomePage = () => {
-  const { noteState } = useNote();
+  const { noteState, noteDispatch } = useNote();
   const { notes } = noteState;
-  console.log(notes)
```
Comment (prajapati-priyanka) [inline src/pages/homepage/HomePage.jsx:8]: remove unwanted consoles.

**Review by prajapati-priyanka — APPROVED** [review]
Functionality is working fine. Good to merge

**Pull request merged**